continuous_integration/docker/base/Dockerfile

```diff
@@ -7,7 +7,7 @@
 #
 FROM centos:7
 
-ARG python_version="3.11"
+ARG python_version="3.13"
```

Comment:
We don't have automation to rebuild these, and they currently fail to rebuild (#837), so lets not change this for now - sorry for the confusion.

I think there was perhaps only the other one we needed to update

Reply:
Ok, no problem, reverted now